config_defaults_inc.php

```diff
@@ -1154,15 +1154,19 @@
 $g_complete_date_format = 'Y-m-d H:i T';
 
 /**
- * jscalendar date format string
- * go to http://www.php.net/manual/en/function.date.php
- * for detailed instructions on date formatting
+ * jquery-ui datepicker date format string
+ * @global string $g_calendar_js_date_format
+ */
+$g_calendar_js_date_format = 'yy-mm-dd';
```

Comment:
The changes in the configuration options here should be reflected in the manual as well.  I also wonder whether we should be changing the semantics of existing configuration options or in this case deprecate this one in obsolete.php and add $g_calendar_date_format and $g_calendar_time_format.  Let's see more opinions.